src/server-types.ts

```diff
@@ -109,6 +109,8 @@ interface DisplayDefinition {
   label: string;
   /** Description to display to the user. */
   description: string;
+  /** Category of the Component. */
+  category?: string;
```

Comment:
Might a component have multiple categories?

Reply:
I asked the same thing and was told no.